packages/@aws-cdk/aws-stepfunctions-tasks/lib/ecs/run-task.ts

```diff
@@ -62,6 +62,15 @@ export interface EcsRunTaskProps extends sfn.TaskStateBaseProps {
    * @see https://docs.aws.amazon.com/AmazonECS/latest/developerguide/launch_types.html
    */
   readonly launchTarget: IEcsLaunchTarget;
+
+  /**
+   * Specifies whether to propagate the tags from the task definition or the service to the tasks in the service
+   *
+   * @see https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_RunTask.html#ECS-RunTask-request-propagateTags
+   *
+   * @default PropagatedTagSource.NONE
```

Comment:
This isn't quite accurate. This implies that we're setting this value when no value is provided. Since we are not, this should be a description of the behavior. Something like:
```suggestion
   * @default - No tags are propagated.
```